pom.xml

```diff
@@ -26,14 +26,17 @@
         <java.version>1.8</java.version>
         <poi-version>3.17</poi-version>
         <postgresql.driver.version>42.2.1</postgresql.driver.version>
-        <solr.version>4.10.4</solr.version>
-	<jena.version>2.13.0</jena.version>
-	<log4j.version>2.6.2</log4j.version>
+        <jena.version>2.13.0</jena.version>
+        <log4j.version>2.6.2</log4j.version>
         <slf4j.version>1.7.22</slf4j.version>
         <jackson.version>2.8.11</jackson.version>
         <jersey.version>2.26</jersey.version>
         <hibernate.version>5.2.8.Final</hibernate.version>
         <hibernate-validator.version>5.4.2.Final</hibernate-validator.version>
+        <!-- PIN Solr to 7.3.x until SOLR-12858 is fixed. This bug affects all integration tests that use Solr
+             https://issues.apache.org/jira/browse/SOLR-12858 -->
+        <solr.client.version>7.3.1</solr.client.version>
+        <solr.server.version>4.10.4</solr.server.version>
```

Comment:
@mwoodiupui : This line is confusing.  Is there a reason we have an old `solr.server.version` specified here?  It looks like this is unused and can be completely removed.  But, maybe I'm missing something?

Reply:
You are correct:  it is unused, and it is confusing.  I had already made a note to remove it in a subsequent PR, but I'll do that now.